.github/workflows/run-ui-tests.yaml

```
@@ -0,0 +1,133 @@
+                  CYPRESS_INCLUDE_TAGS: "${{ inputs.test_tags }}"
+                  CYPRESS_baseUrl: "${{ env.UI_URL }}"
+                  CYPRESS_git_user: "fakeuser"
+                  CYPRESS_git_password: "${{ secrets.GITHUB_TOKEN }}"
+                  CYPRESS_git_key: "${{ secrets.GITHUB_TOKEN }}"
+              with:
+                  working-directory: tackle-ui-tests
+
+            - uses: actions/upload-artifact@v4
+              if: failure()
+              env:
+                  AUTH_KIND: ${{ fromJSON('{true:"auth", false:"noauth"}')[inputs.feature_auth_required] }}
```

actionlint (1.7.7)

broken JSON string is passed to fromJSON() at offset 2: invalid character 't' looking for beginning of object key string

(expression)